bcda/cli.go

```diff
@@ -108,15 +111,20 @@ func importCCLF0(fileMetadata cclfFileMetadata) (map[string]cclfFileValidator, e
 		log.Error(err)
 		return nil, err
 	}
+	fmt.Printf("Imported CCLF0 file %s.\n", fileMetadata.name)
+	log.Infof("Imported CCLF0 file %s.", fileMetadata.name)
+
 	return validator, nil
 }
 
 func importCCLF8(fileMetadata cclfFileMetadata) error {
 	fmt.Printf("Importing CCLF8 file %s...\n", fileMetadata.name)
 	log.Infof("Importing CCLF8 file %s...", fileMetadata.name)
 
-	if fileMetadata.filePath == "" {
-		return errors.New("file path must be provided")
+	if (cclfFileMetadata{}) == fileMetadata {
+		err := errors.New("File CCLF8 not found")
```

Comment:
Golang recommends not capitalizing error strings https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
oooo good catch!